addons/purchase/models/purchase.py

```diff
@@ -372,7 +372,19 @@ def print_quotation(self):
         self.write({'state': "sent"})
         return self.env.ref('purchase.report_purchase_quotation').report_action(self)
 
+    def _approval_allowed(self):
```

Comment:
Could you put this part at the end of the file with the other 'private functions' ? (see [Conventions](https://www.odoo.com/documentation/14.0/reference/guidelines.html#symbols-and-conventions)) 
Thanks

Reply:
Done